compiler/record-hessian/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+file(GLOB_RECURSE SOURCES "src/*.cpp")
+
+add_library(record-hessian STATIC ${SOURCES})
+
+target_include_directories(record-hessian PUBLIC include)
+target_include_directories(record-hessian PRIVATE src)
+
+target_link_libraries(record-hessian luci_import)
+target_link_libraries(record-hessian luci_env)
+target_link_libraries(record-hessian luci_export)
+target_link_libraries(record-hessian luci_interpreter)
+target_link_libraries(record-hessian luci_log)
+target_link_libraries(record-hessian dio_hdf5)
+
+install(TARGETS record-hessian DESTINATION lib)
+install(DIRECTORY include/ DESTINATION include
+        FILES_MATCHING PATTERN "*.h")
+
+if(NOT ENABLE_TEST)
+  return()
+endif(NOT ENABLE_TEST)
+
+nnas_find_package(GTest REQUIRED)
+
+file(GLOB_RECURSE TESTS "tests/*.test.cpp")
```

Comment:
do you have particular reason to put test files in `tests` folder?
can you put them in `src` like other modules?